### PR TITLE
optional: disabling scan to access directly on upload phase

### PR DESCRIPTION
**Pull request by malys**
Hello,
I add two options:
* scan files only
* upload without scan
For massive upload, I schedule scan files only every week but I upload photos after opening session (I'm windows user).

Python2 walk files have poor performance. In ths way, I save time.

I'm not python developer but to my mind, for the next version, I think that it will be a good idea to remove all tables except *allMedia*  and add it attributes like 'bad' ...

Thank you for your help.

### CODE REVIEW

**Comment by oPromessa** [comment]
Hi @malys, thanks for your pull request. I'm struggling myself with the long file scan. 

Side note: With feature #33 I've reduced about 10 minutes on a Synology system (from 1hour to 50minutes) the processing of 61000 pics bypassing the checking if the file is still on flickr.

Quick question for me to better understand these features.
- (scan option) saving the file structure into the DB would save you time next (and each) time you run. Good point. Got it!
- (upload without scan) avoid scanning the file structure you are going to be able to:
   - basically check for any file changes and replace them on flickr
   - but you will not detect any new files... until you run the scan option again, of course. 
That's reasonable... you don't change files everyday! But then, one could simply run uploadr once a week.
So my question is how to you deal with "detecting new files", manually maybe?

Thanks for your support!

**Comment by malys** [comment]
In my case, one time a week is sufficient because it's my current frequency to copy picture from my cam to my computer.
My scheduled task (scan option) will clean table , scan folders and populate table with uploaded files, updates files and new files. After that, (upload without scan) will upload new files and updated files

**Comment by oPromessa** [comment]
@malys checkout version 2.6.7 which is much faster for runs after the 1st one. Calls to md5Checksum (time consuming) are now much more limited while keeping the "scan files". I think it will end up addressing your question.
